api/wallet_api-v0.yml

```diff
@@ -99,19 +99,20 @@ paths:
           description: Internal Server Error - The server has encountered a situation it does not know how to handle.
           schema:
             $ref: "#/definitions/Error"
+  /rest/wallet/import/{nickname}:
     put:
       operationId: restWalletImport
+      produces:
+        - application/json   
       parameters:
-        - in: body
-          name: body
+        - in: path
+          name: nickname
+          type: string
           required: true
-          x-nullable: false
-          schema:
-            $ref: "#/definitions/Wallet"
-      produces:
-        - application/json
+          x-nullable : false
+          description: Wallet nickname.
       responses:
-        "204":
+        "200":
```

Comment:
If tehir is not content, it should be a 204 - no content.

Reply:
it returns 	return operations.NewRestWalletCreateOK().WithPayload(
		&models.Wallet{
			Nickname: newWallet.Nickname,
			Address:  newWallet.Address,
			KeyPair: models.WalletKeyPair{
				PrivateKey: privK,
				PublicKey:  pubK,
				Salt:       salt,
				Nonce:      nonce,
			},
		}) so there is a body, do you put 200 or do we send an empty response ?